### ISSUE TITLE
Can my desk use 3 legs?

### ISSUE THREAD

**Issue by petersenna** [trainman419/bekant#9]
Hi I am looking into upgrading my desk, but I need 3 legs. I am considering buying 4 legs, but use only 3 of them. Problem is I could not find out if the protocol work with only one master and two slaves, or if I can have 3 slaves as well? Thank you!

**Comment by petersenna** [comment]
Err I guess I got my answer on wikipedia: `LIN is a broadcast serial network comprising 16 nodes (one master and typically up to 15 slaves)`. So I guess it is likely to work with up to 15 legs :smile:

**Comment by trainman419** [comment]
I would be cautious here. I think LIN assumes that each device has a unique address and I think the Bekant legs have some protocol extension to assign unique addresses to each leg on startup. I'm not sure if that protocol extends to assigning unique address for more than two legs.

Even if the legs support it, you will probably need to spend some time porting over the initialization code ( see #2 ) and then understand it well enough to try to extend it to 3 legs.

**Comment by gritstub** [comment]
@petersenna : Did you ever solve this issue? I am trying to set up a Bekant with 3 or 4 legs.

**Comment by petersenna** [comment]
> @petersenna : Did you ever solve this issue? I am trying to set up a Bekant with 3 or 4 legs.

@gritstub I went on a different direction. I kept my static table.

To stand, I use monitor arm that has enough range for sitting and standing, and I use an Medical Adjustable Overbed Bedside Table for the wireless keyboard and mouse.

Notice the long pole: https://www.digitec.ch/en/s1/product/ergotron-table-mounting-arm-table-46-monitor-mount-4656121
 
https://www.amazon.ca/Medical-Adjustable-Overbed-Bedside-Hospital/dp/B00QMNTMXA
